src/main/modOrganizer.ts

```diff
@@ -2,11 +2,34 @@ import path from "path";
 import childProcess from "child_process";
 import { USER_PREFERENCE_KEYS, userPreferences } from "@/main/config";
 import { logger } from "@/main/logger";
+import { copyGameFiles, checkGameFiles } from "@/main/gameFiles";
+import { copyEnbFiles, checkEnbFiles } from "@/main/enb";
 
 export const MO2EXE = "ModOrganizer.exe";
 
-export function launchGame() {
+export async function launchGame() {
   try {
+    const gameFilesExist = await checkGameFiles(
```

Comment:
Maybe we could add a log here so we know we've started the process of copying files
```suggestion
    logger.info('Copying game files on launch');
    const gameFilesExist = await checkGameFiles(
```